src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/coding/ReturnCountCheckExamplesTest.java

```diff
@@ -19,12 +19,12 @@
 
 package com.puppycrawl.tools.checkstyle.checks.coding;
 
-import org.junit.jupiter.api.Disabled;
+import com.puppycrawl.tools.checkstyle.checks.coding.returncount.MSG_KEY;
+import com.puppycrawl.tools.checkstyle.checks.coding.returncount.MSG_KEY_VOID;
```

Comment:
you need to put `static` modifier and make import to be correct - with class name.
look for examples in already merged PRs: https://github.com/checkstyle/checkstyle/commit/7123cc2b7ca95f5f349e54776a02724f2e7b0d69#diff-8b9ed11f4d5073c3d06414a4a2e976ff103dfad4e90cac65009cfa7648cb28b2R22